system/diagnostic_graph_aggregator/CMakeLists.txt

```diff
@@ -29,6 +23,17 @@ ament_auto_add_executable(plantuml
 )
 target_include_directories(plantuml PRIVATE src/common)
 
+ament_auto_add_library(aggregator SHARED
```

Comment:
It is better to use `${PROJECT_NAME}` according to [this page](https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/directory-structure/#exporting-a-composable-node-component-executables).
 It also makes it easier to add `autoware_` prefix.